crates/gpui/src/text_system/font_features.rs

```diff
@@ -142,3 +142,18 @@ impl schemars::JsonSchema for FontFeatures {
         schema.into()
     }
 }
+
+fn is_valid_feature_tag(tag: &str) -> bool {
+    if tag.len() != 4 {
+        return false;
+    }
+    for ch in tag.chars() {
+        match ch {
+            '0'..='9' => continue,
```

Comment:
I think we can replace this `match` with `ch.is_ascii_alphanumeric`: https://doc.rust-lang.org/std/primitive.char.html#method.is_ascii_alphanumeric

Reply:
It's surprising that Rust already has this function.